pom.xml

```diff
@@ -16,12 +16,44 @@
     <module>hefquin-service</module>
   </modules>
 
+  <!-- Project Meta Data -->
+  <name>HeFQUIN</name>
+  <description>HeFQUIN is a query federation engine for heterogeneous federations of graph data sources, including federations of knowledge graphs.</description>
+  <url>http://github.com/LiUSemWeb/hefquin</url>
+
+  <!-- License -->
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <!-- Developer Information -->
+  <developers>
+    <developer>
+      <name>Olaf Hartig</name>
+      <email>olaf.hartig@liu.se</email>
+      <organization>Linköping University</organization>
+      <organizationUrl>https://liu.se/</organizationUrl>
+    </developer>
+  </developers>
+
+  <!-- SCM (Source Code Management) -->
+  <scm>
+    <connection>scm:git:git://github.com/LiUSemWeb/HeFQUIN.git</connection>
+    <developerConnection>scm:git:ssh://github.com:LiUSemWeb/HeFQUIN.git</developerConnection>
+    <url>http://github.com/LiUSemWeb/HeFQUIN/tree/main</url>
```

Comment:
Is there a reason why you didn't simply put `http://github.com/LiUSemWeb/HeFQUIN`? (I'm just curious)

Also, shouldn't it be `https://` instead of `http://`?

Reply:
No reason other than that one of the reference examples I looked at used it. But as far as I can tell, using the shorter version should be fine. Unless we stumble across some motivation for it I think we should go for the shroter one, looks neater. I've pushed the update.

Indeed it should be `https` instead of `http` (part of the same commit as above).